frontend/www/js/omegaup/components/problem/Form.vue

```diff
@@ -59,6 +59,7 @@
           :validator-time-limit="validatorTimeLimit"
           :valid-languages="data.validLanguages"
           :validator-types="data.validatorTypes"
+          @languages-changed="languagesChanged"
```

Comment:
```suggestion
          :languages.sync="languages"
```

y eliminar L57.

aquí hay que hacer unos cuantos más cambios: hay que renombrar `initialLanguage` por `language` en frontend/www/js/omegaup/components/problem/Settings.vue y como ahora habría un conflicto de nombres, lo que ahora se llama `language` debería llamarse `currentLanguage`.